openedx/core/djangoapps/user_authn/views/tests/test_views.py

```diff
@@ -75,12 +76,27 @@ class UserAccountUpdateTest(CacheIsolationTestCase, UrlResetMixin):
 
     ENABLED_CACHES = ['default']
 
+    def _create_account(self, username, password, email):
+        # pylint: disable=missing-docstring
+        registration_url = reverse('user_api_registration')
+        resp = self.client.post(registration_url, {
+            'username': username,
+            'email': email,
+            'password': password,
+            'name': username,
+            'honor_code': 'true',
+        })
+        self.assertEqual(resp.status_code, 200)
+
     def setUp(self):
         super(UserAccountUpdateTest, self).setUp()
 
         # Create/activate a new account
-        activation_key = create_account(self.USERNAME, self.OLD_PASSWORD, self.OLD_EMAIL)
-        activate_account(activation_key)
+        self._create_account(self.USERNAME, self.OLD_PASSWORD, self.OLD_EMAIL)
+        mail.outbox = []
+        user = User.objects.get(username=self.USERNAME)
+        registration = Registration.objects.get(user=user)
```

Comment:
If you used CreateAccountMixin, could you also move `get_activation_key` to the mixin?